### ISSUE TITLE
File highlight: Toolbar button is registered multiple times

### ISSUE THREAD

**Issue by RunDevelopment** [PrismJS/prism#1817]
**Information:**
- Prism version: latest
- Environment: Browser

**Description**
The [File highlight plugin](https://prismjs.com/plugins/file-highlight/) registers the 'Download this file' every time [`fileHighlight`](https://github.com/PrismJS/prism/blob/master/plugins/file-highlight/prism-file-highlight.js#L84) is executed. 
But [Toolbar](https://prismjs.com/plugins/toolbar) [can't deal with registering multiple callbacks on the same key](https://github.com/PrismJS/prism/blob/master/plugins/toolbar/prism-toolbar.js#L46).

**Possible solutions**

1. Adjust File highlight to register the button once and only once.
2. #1816 

Regardlessly, we should add a check to Toolbar so that you can't register on the same key twice. (#1818)

**Comment by atomize** [comment]
I actually just noticed this as well while messing with my fork from the file-highlight enhancement, but I was having a hard time replicating it as it seemed to only happen on my local dev server that was auto-reloading and I chalked it up to that.

 I could take a look in to this and report back. Also, I think that your #1816 suggestion is the best, as the 'download button' seems a bit out of scope with the fileHighlight plugin, especially since it depends on ANOTHER separate plugin (toolbar). It would seem reasonable for both the simplicity of the file-highlight plugin, as well as for ease of debug and integration, that suggestion #1816 be taken in to consideration. Just my 2 cents.
